save-api/src/main/kotlin/com/saveourtool/save/api/github/Project.kt

```diff
@@ -0,0 +1,33 @@
+package com.saveourtool.save.api.github
+
+import java.net.URL
+
+data class Project(
```

Comment:
May be rename to something like `GitHubProject` so as not to overlap over existing `Project` enitity?

Reply:
Already done that in the newest version.